cli/lib/kontena/cli/helpers/exec_helper.rb

```diff
@@ -96,6 +96,12 @@ def websocket_exec_write(ws, msg)
     def websocket_exec_write_thread(ws, tty: nil)
       Thread.new do
         begin
+          if tty
+            console_height, console_width = IO.console.winsize
```

Comment:
This does not fix #2601then. Edit that to only cover the initial terminal size, and I can create a new issue for the resize?